test/autotests/http/github-repo-enabled/invalid/test.js

```diff
@@ -0,0 +1,11 @@
+exports.description = 'should return enabled false when a repo does not exist'
+
+exports.buildEndpoint = function (t) {
+  const { httpServerPort } = t.context
+  return `:${httpServerPort}/v1/repository/github/some-random-user/some-random-repo/enabled`
```

Comment:
This is still a valid url/input, so I'd expect the file/path descriptive to be 'disabled' rather than 'invalid'.

Also, might be worth appending a uuid to the user just to safeguard against someone unintentionally colliding with this (all future tests should only create user/repos with uuids, but just to be extra paranoid)